src/Initializer/InputAux.hpp

```diff
@@ -103,6 +103,59 @@ void convertStringToMask(const std::string& stringMask, ContainerT& mask) {
   }
 }
 
+/**
+ * \brief Converts an input string to a vector of the datatype T. If the input string does not contain enough
+ * items to convert to an array of length n, we fill with default values. If the input string contains
+ * more than n items, we only consider the first n ones.
+ *
+ * \throws runtime_error if the input string contains parameters, which can not be converted to T.
+ * */
+template<typename T, size_t n>
+std::array<T, n> convertStringToArray(const std::string& inputString) {
+  auto result = std::array<T, n>();
+  if (inputString.empty()) {
+    return result;
+  }
+
+  size_t begin = 0;
+  size_t wordCount = 0;
+  enum class State { Word, Delimiter };
+  State s = inputString.at(0) == ' ' ? State::Delimiter : State::Word;
```

Comment:
This should be in a const variable "delimiter"

Reply:
It's now a function argument.